Source/hu_stuff.c

```diff
@@ -1088,11 +1090,10 @@ void HU_Drawer(void)
       if (crispy_hud)
       {
           ST_Drawer (false, true);
-          if (hud_timests)
-          {
+          if (1 & hud_timests)
```

Comment:
Sorry for nit-picking, but could you reverse this, please? Like `hud_timests & 1`. Bonus points for introducing an `enum` to hold thd calues, e.g. `TIMESTS_TIME =1`.

Reply:
Wasn't too sure where to put these enums, so I put them right beneath the normal HUD stuff